client/src/components/Overview.jsx

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
+import StarRating from './OverviewComponents/StarRating.js';
+import Reviews from './OverviewComponents/Reviews.js';
+import axios from 'axios'
+import {token} from '../../../config.js'
+import StyleSelector from './OverviewComponents/StyleSelector.js'
+// import ImageGallery from '/Users/danielghaly/Desktop/Hack Reactor/fec3/client/src/components/OverviewComponents/ImageGallery.js'
+import AddToCart from './OverviewComponents/AddToCart.js'
+
+
+class Overview extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      numberOfReviews: null,
+      rating: null,
+      productName: null,
+      productCategory: null,
+      productPrice: null,
+      thumbnails: [],
+      styles: [],
+      selectedStyle: 0,
+      sizes: [],
+      outOfStock: null,
+      selectedSize: null,
+      quantity: null,
+      results: null,
+      validATC: null
+
+    }
+  }
+
+  componentDidMount() {
+
```

Comment:
make sure to turn on a linter so all these unnecessary spaces are fixed in the future